src/Umbraco.Web.UI.Client/src/packages/core/repository/repository-details.manager.ts

```diff
@@ -0,0 +1,239 @@
+import type { UmbDetailRepository } from './detail/detail-repository.interface.js';
+import type { UmbControllerHost } from '@umbraco-cms/backoffice/controller-api';
+import { UmbArrayState } from '@umbraco-cms/backoffice/observable-api';
+import { type ManifestRepository, umbExtensionsRegistry } from '@umbraco-cms/backoffice/extension-registry';
+import { UmbExtensionApiInitializer } from '@umbraco-cms/backoffice/extension-api';
+import { UmbControllerBase } from '@umbraco-cms/backoffice/class-api';
+import { UMB_ACTION_EVENT_CONTEXT } from '@umbraco-cms/backoffice/action';
+import { UmbEntityUpdatedEvent } from '@umbraco-cms/backoffice/entity-action';
+
+interface UmbRepositoryStatus {
```

Comment:
I think this interface name needs to be more explicit 🤔 Something in the lines of `UmbRepositoryDetailRequestStatus`?